core/labeled-replies-3.3.md

```diff
@@ -0,0 +1,36 @@
+---
+title: IRCv3.3 Labeled replies
+layout: spec
+copyrights:
+  -
+    name: "Alexey Sokolov"
+    period: "2015"
+    email: "alexey-irc@asokolov.org"
+---
+Name of this capability is `labeled-replies`.
+If client requests `labeled-replies`, it MUST also request `batch` capability.
+
+If the capability is enabled, client MAY tag its messages with `label` tag.
+
+For messages from client which are tagged with `label`, server MUST send exactly
+one message tagged with the same `label` value in response to such message.
+If response contains more than one message, `batch` can be used to group them.
+In that case start of the batch MUST be tagged with the label, and the batch type MUST be `labeled-response`.
```

Comment:
As discussed below, maybe something like:

In that case start of the batch MUST be tagged with the label. The batch type MUST be an existing batch type if it is applicable to the entire response set, or `labeled-response` if one is not.